code/game/objects/items/inspector.dm

```diff
@@ -93,6 +97,49 @@
 	else
 		. += "\The [cell] is firmly in place. [span_info("Ctrl-click with an empty hand to remove it.")]"
 
+/obj/item/inspector/interact_with_atom(atom/interacting_with, mob/living/user, list/modifiers)
+	. = ..()
+	if(!interacting_with.can_interact(user, TRUE))
+		return
```

Comment:
You shouldn't *need* to run `can_interact` here? This proc is only called if we're clicking on this atom, so it already checks most of the generic stuff (incapacitated for example). Unless the sole reason is to weed out telekinesis users, which can probably be done better with just an `Adjacency` check